kt/kt.go

```diff
@@ -9,11 +9,36 @@ import (
 	"net"
 	"net/http"
 	"net/url"
+	"path"
 	"strconv"
+	"sync"
 	"sync/atomic"
 	"time"
+
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+
+	"github.com/prometheus/client_golang/prometheus"
 )
 
+var certExpiryTimestamp = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Name: "ktrpc_client_certificate_expiry_timestamp",
+	Help: "The certificate expiry timestamp (UNIX epoch UTC) labeled by the certificate serial number",
+},
+	[]string{
+		"serial",
+	},
+)
+
+var once sync.Once
+
+func init() {
+	once.Do(func() {
```

Comment:
This is unnecessary, `init()` will ever only be called once. And you can avoid this altogether with https://godoc.org/github.com/prometheus/client_golang/prometheus/promauto.

Reply:
Thanks, chris! Pushed a fix for Geoffreys PR.